assets/xml/objects/object_ik.xml

```diff
@@ -55,52 +54,52 @@
         <Texture Name="object_ik_TLUT_010A30" OutName="tlut_010A30" Format="rgba16" Width="16" Height="16" Offset="0x10A30" />
         <Texture Name="object_ik_Tex_010C30" OutName="tex_010C30" Format="ia16" Width="16" Height="16" Offset="0x10C30" />
         <Texture Name="object_ik_Tex_010E30" OutName="tex_010E30" Format="ci8" Width="16" Height="16" Offset="0x10E30" />
-        <Texture Name="object_ik_Tex_010F30" OutName="tex_010F30" Format="ci8" Width="16" Height="16" Offset="0x10F30" />
+        <Texture Name="gAxePomelJewel1Tex" OutName="axe_pomel_1_jewel_tex" Format="ci8" Width="16" Height="16" Offset="0x10F30" />
```

Comment:
Since these symbols are global they all need the `IronKnuckle` part. There's also still some unnamed stuff in here... is it unused or leftover from OoT or something?